### ISSUE TITLE
Drop `from_python` from `BaseConstant`

### ISSUE THREAD

**Issue by vpetrovykh** [edgedb/edgedb#1145]
We use `from_python` mainly to convert `schema.Object.field` to `edgeql.AST.field`.  Since fields are typed anyway, we don't really need to have the dispatch in `BaseConstant`, we can just reimplement it in `AlterObjectProperty._get_ast` using the field type information.  GraphQL can do it's own thing.  We don't have constant folding other than string concatenation, so it's not an issue for that use case either.

Bottom line: the proper action is to kill `BaseConstant.from_python`.

_Originally posted by @elprans in https://github.com/edgedb/edgedb/pull/1131_
